### PR TITLE
fix: tagged username in chat showed name + user id after sending message(#3168)

### PR DESCRIPTION
**Pull request by staticGuru**
.closes tagged username in chat showed name + user id after sending message(#3168) #3168

### CODE REVIEW

File: packages/app-extension/src/components/common/Account/MnemonicInput.tsx [hunk 1]
```diff
@@ -178,8 +188,27 @@ export function MnemonicInput({
               : "Enter your 12 or 24-word secret recovery mnemonic to add an existing wallet."}
           </SubtextParagraph>
         </Box>
+        <MnemonicTextInput
```
Comment (hkirat) [inline packages/app-extension/src/components/common/Account/MnemonicInput.tsx:191]: Could you explain the reason for the changes in this file?

Comment (staticGuru) [reply packages/app-extension/src/components/common/Account/MnemonicInput.tsx:191]: @hkirat 
It's kind of new feature,
I already created PR for the feature.(#3099)
#3099

Comment (staticGuru) [reply packages/app-extension/src/components/common/Account/MnemonicInput.tsx:191]: @hkirat 
kindly verify the #3099 PR also.
It is single mnemonic input field to user's can easily type the mnemonic phrase.

File: packages/chat-sdk/src/components/Message.tsx [hunk 2]
```diff
@@ -909,7 +921,7 @@ function SecureTransferElement({
 export function ChatMessages() {
   const { chats, userId } = useChatContext();
   const theme = useCustomTheme();
-
+  console.log("chatss", chats);
```
Comment (yash-w3) [inline packages/chat-sdk/src/components/Message.tsx:924]: might want to remove this

**Comment by yash-w3** [comment]
@staticGuru try to branch out from the latest master when you create a new PR, it will help in reducing the scope to the particular issue or feature. As of now the PRs has multiple features and issues in same branch

**Comment by staticGuru** [comment]
@yash-w3 
I create the new PR, based on your suggestions. #3177

**Comment by staticGuru** [comment]
@hkirat kindly verify the new features in #3099 
I suggest you review the features and let me know.

**Pull request closed without merging**